doc/tutorials/dune-package-management/setup.md

```diff
@@ -0,0 +1,65 @@
+# Setting up Package Management with Dune
+
+The idea of package management with Dune has been to be as unobtrusive as
+possible. Thus most projects can easily be built with just the minimum of
+changes.
+
+In this tutorial we'll create a simple project to use the OCaml package
+management for the very first time.
+
+## Declare dependencies
+
+The best way to work with the package management is to declare your
+dependencies in the `dune-project` file, like so:
+
+```dune
+(lang dune 3.17)
+(name test)
+
+(generate_opam_files true)
+
+(package
+  (name test)
+  (depends
```

Comment:
We should explain that dependencies go in `depends` and what the format is with a examples – I've seen enough people fumble with this already.

Some examples would be:

* For the latest version, just use the package name: `cmdliner`
* For an exact version, use `(cmdliner (= 1.2.3))`
* For a constraint, use `(cmdliner (>= 1.2.0))`

Reply:
I think that's a good point! I've added a section on this in a later part.